synapse/api/auth.py

```diff
@@ -775,15 +775,24 @@ def check_in_room_or_world_readable(self, room_id, user_id):
             )
 
     @defer.inlineCallbacks
-    def check_auth_blocking(self):
+    def check_auth_blocking(self, user_id=None):
```

Comment:
I wonder if we should call this `check_server_limits` or something

Reply:
I think I agree, since it will be a change in a lot of places I'll submit a separate PR

Reply:
err given I have a few outstanding, I'll wait until a few more of them land